definitions/scenarios/upgrade_to_foreman_nightly.rb

```diff
@@ -56,7 +56,7 @@ def compose
         modules_to_enable = ["foreman:#{el_short_name}"]
         add_step(Procedures::Packages::EnableModules.new(:module_names => modules_to_enable))
       end
-      add_step(Procedures::Packages::Update.new(:assumeyes => true))
+      add_step(Procedures::Packages::Update.new(:assumeyes => true, :clean_cache => false))
```

Comment:
I think this one should clean cache, as there is no dedicated `download_only` step (and we'd need to implement that for debian if we wanted it here)